client/components/ErrandTracker/BottomSheet/BottomSheet.jsx

```diff
@@ -8,17 +8,19 @@ import {
 import MessageButton from './MessageButton';
 import Courier from './Courier';
 
-const BottomSheet = ({
-  eta, errandRunner, errandName, chatId,
-}) => (
-  <View style={styles.container}>
-    <Text>{errandName}</Text>
-    <Text>ETA: {Math.floor(eta)} MINUTES </Text>
-    <Courier />
-    <MessageButton chatId={chatId} />
-  </View>
-);
+const BottomSheet = ({ eta, errand }) => {
+  const { errandName, chatId, status } = errand;
 
+  return (
+    <View style={styles.container}>
+      <Text>{errandName}</Text>
+      <Text>ETA: {Math.floor(eta)} MINUTES </Text>
+      <Text>Status: {status} </Text>
+      <Courier errand={errand} />
+      <MessageButton chatId={chatId} />
```

Comment:
Should work now. Think there was a `navigate={navigate}` or two missing lol I keep forgetting to add those.

Reply:
Yaaay ty